_docs/first-steps.md

```diff
@@ -0,0 +1,126 @@
+
+# Interacting with a registry and Portus in the CLI
+
+## Log in and out
+
+In order to push or pull any image to your registry you will have to login
+first with docker. Assuming that your registry's hostname is
+`https://my.registry:5000`, then you should perform the following in a terminal:
+
+    $ docker login my.registry:5000
+
+The previous command will ask for your credentials and, if everything goes as
+expected, you will successfully login to your registry. Note two things:
+
+1. If you don't specify a hostname, then Docker assumes the
+   [Docker Hub](https://hub.docker.com/).
+2. You don't have to specify the `https://` part.
+
+After logging in your credentials will be stored in your system so you won't
+have to login again unless you explicitly log out with the `docker logout`
+command. Note that in the `logout` command you will also have to pass the
+hostname (or docker will assume Docker Hub).
+
+## Push & pull
+
+The Docker Engine will allow you to push or pull only if you are logged in. If
+this is the case, then it will be up to the authorization service (Portus in
+our case) to decide whether you can push or pull. This depends on which team
+your user belong, which role plays in said team and so on. Check out [this
+page](/features/3_teams_namespaces_and_users.html) for more information on the
+topic.
+
+Assuming that your user is called `mssola`, you can be sure that it exists a
+namespace also called `mssola`. This is because each user has a **personal
+namespace**. In there, you will always be able to push and pull. We will use
+this as an example because of its simplicity.
+
+Let's say that you want to push a local image called `image:latest` to your
+registry. The first thing that comes to our mind is to perform the following
+*wrong* command:
+
+    $ docker push image:latest
+
+This is wrong because docker will assume that your image should be pushed onto
+the Docker Hub. In order to avoid this you have to specify the hostname of your
+registry in the name of the image that you want to push. In order to
+accomplish this, we will use the `docker tag` command:
+
+    $ docker tag image:latest my.registry:5000/mssola/image:latest
+
+Note two things in the previous command:
+
+1. The `https://` part is not specified.
+2. We also added `mssola` to it. We did this to specify that the `image:latest`
+   image should be stored inside of the `mssola` namespace, which is your
+   personal namespace. If you don't provide this, then Portus will assume that
+   you want to push this image into the **global namespace**, which is only
+   writeable by administrators. You can read more about this
+   [here](/features/3_teams_namespaces_and_users.html).
+
+Now you can push it:
+
+    $ docker push my.registry:5000/mssola/image:latest
+
+If everything goes as expected, your image should have been pushed into your
+registry! Now, whenever you want to pull the image down again, just perform the
```

Comment:
I'm not sure whether or not "pull … down" is correct in this context.  I'd just simply go with "pull".